### PR TITLE
add sphere viz support

### PR DESCRIPTION
**Pull request by simonbyrne**
Not sure exactly where this should go, but the interactive HTML output is pretty nice.

### CODE REVIEW

**Comment by simonbyrne** [comment]
bors try

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -279,6 +279,17 @@ steps:
       queue: central
       slurm_ntasks: 1
 
+  - label: ":computer: Visualization on a sphere"
+    key: "cpu_sphere_viz"
+    command:
+      - "julia --color=yes --project=examples/sphere examples/sphere/viz.jl"
+    artifact_paths:
+      - "examples/sphere/output/viz/*"
```
Comment (charleskawczynski) [inline .buildkite/pipeline.yml:287]:
```suggestion
      - "examples/sphere/output/*"
```

**Comment by simonbyrne** [comment]
bors try

**Comment by charleskawczynski** [comment]
It even works nicely from my phone! ❤️

**Comment by simonbyrne** [comment]
It turns out the S3 links aren't stable, but the buildkite redirects are, so i fixed the above link.

**Pull request closed without merging**